.gitignore

```diff
@@ -19,3 +19,7 @@ MANIFEST-*
 OPTIONS-*
 CURRENT
 LOCK
+
+# standard file naming when generating benchmarks through "go ppof"
+mem*.out
+cpu*.out
```

Comment:
nit: `.git/info/exclude` can be used to ignore personal files. I also do this for a folder named `tmp` to put my personal files and notes related to the repo there.